File: data/c/pattern_18/code.c
#include <stdio.h>

int main(void)
{
    int num;
    printf("Enter the number of rows and columns for the square: ");
    scanf("%d", &num);

    for (int i = 1; i <= num; i++)
    {
        for (int j = 1; j <= num; j++)
        {
            printf("%d ", (i + j) % 2);
        }
        printf("\n");
    }

    return 0;
}

File: data/c#/pattern_5/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = num; i >= 1; i--)
        {
            for (int j = num; j >= 1; j--)
            {
                Console.Write(j + " ");
            }
            Console.Write("\n");
        }
    }
}
